homeworks/Aleksandr.Turuta_turuta7/homework_7/homework_7.js

```diff
@@ -0,0 +1,19 @@
+function fibonacci(n) {
+    if (n === 1 || n === 2) return 1;
+    if (n === 0) return 0;
+    return fibonacci(n - 2) + fibonacci(n - 1);
+}
+
+function cycleAndTest(rand) {
```

Comment:
function naming needs to be refactored... `fibonacci` does the number check and `cycleAndTest` solves the actual task... a bit confusing

also, it may not even require two separate functions